platforms/tktrex/extension/webpack.config.ts

```diff
@@ -67,7 +48,7 @@ const { buildENV, ...config } = getExtensionConfig(
     distDir: PATHS.DIST,
     manifestVersion: APP_VERSION,
     transformManifest: (m) => {
-      if (NODE_ENV === 'development') {
+      if (process.env.NODE_ENV === 'development') {
```

Comment:
considering we're not hard-checking "development", I would change this with a boolean at the beginning of the file so to be sure that ambiguous condition can happen.